### DISCUSSION TITLE
Is it possible to start from center?

### DISCUSSION THREAD

**Discussion by enjoycod1ng** [matteobruni/tsparticles#1959] [Q&A] [answered]
I noticed that sometimes particles appeared from each corner and expand to the whole canvas.

It looks good but is there any way to start from center not corner?

**Comment by matteobruni** [comment, marked as answer]
Hello @mastercodingbear,

You can place an emitter in the center of the canvas for an effect like this: https://codepen.io/matteobruni/pen/OJpQvdO

**Reply by enjoycod1ng** [reply]
Thanks a lot. That's what i'm looking for.

**Reply by enjoycod1ng** [reply]
I tried to place emitter but it seems to don't work.

Could you show me another example with particle(parallax preset in demo)which starts from center and expanding ?

**Reply by matteobruni** [reply]
Can you share your configuration? So I can try it to see if there are some issues.

Anyway other samples are here: 

https://github.com/matteobruni/tsparticles/blob/main/demo/vanilla/public/presets/

like this one

https://github.com/matteobruni/tsparticles/blob/main/demo/vanilla/public/presets/hyperspace.json

**Reply by enjoycod1ng** [reply]
I like the `parallax preset` but at the first time all dots appear in center of the canvas and they expand the whole screen and start normal animation.
starts like the example you showed me.
Does it make sense?

**Reply by matteobruni** [reply]
> I like the `parallax preset` but at the first time all dots appear in center of the canvas and they expand the whole screen and start normal animation.
> starts like the example you showed me.
> Does it make sense?

I didn't understand sorry. The parallax preset is not using emitters, it's easier to start from a centered emitter preset and add the interactivity hover options to that.

Anyway, screenshots, video recordings or the configuration are more helpful than any description since I can easily figure out what's wrong.

**Reply by enjoycod1ng** [reply]
Ok thanks. Just another problem. Can I use emitter with images?

**Reply by matteobruni** [reply]
Yes, all shapes can be used everywhere.

You can see a sample of multiple shapes (also images) used in an emitters here: https://codepen.io/matteobruni/pen/KKmKaVb

**Reply by enjoycod1ng** [reply]
It's so amazing module. Thanks for your help again.

**Reply by enjoycod1ng** [reply]
![Screenshot 2021-08-11 024517](https://user-images.githubusercontent.com/84164835/128949243-6cf1f518-c279-41d4-a787-f3407998e821.png)
**start of animation**

![Screenshot 2021-08-11 024512](https://user-images.githubusercontent.com/84164835/128949253-27336b40-3bdf-4a62-ab3d-0e10bd0e97d4.png)
**normal animation**

I used the emitters to start the animation. 
The first image is start image of the animation, after pushing all particles I want to stop and set the normal particle animation.

Almost work but not responsive and if when changing the window size particles disappear.

```
emitters: {
  direction: "none",
  position: {
    x: 50,
    y: 50
  },
  rate: {
    quantity: 4,
    delay: 2
  },
  life: {
    count: 1,
    delay: 0.5,
    duration: 
  }
},
particles: {
    color: {
      value: 'rgba(130, 195, 255, 0.58)'
    },
    number: {
      density: {
        enable: true,
        area: 800,
        factor: 1000
      },
      value: 0
    },
    size: {
        value: 4
    },
    links: {
      color: {
        value: 'rgba(130, 195, 255, 0.58)'
      },
      opacity: 0.58,
      triangles: {
        enable: true,
        frequency: 0.1,
      }
    },
    move: {
      speed: 0.4
    }
},
interactivity: {
    events: {
        onhover: {
            enable: true,
            mode: 'grab',
        },
        resize: true
    }
},
shape: {
  type: [
    "circle"
  ]
}
```

[screencast-translate.google.com-2021.08.11-02_55_09.zip](https://github.com/matteobruni/tsparticles/files/6965033/screencast-translate.google.com-2021.08.11-02_55_09.zip)
attached video file what I have done and ran into.

**Reply by enjoycod1ng** [reply]
Hi.
This is what I'm talking about particles appearing.

[screencast-translate.google.com-2021.08.11-09_17_18.zip](https://github.com/matteobruni/tsparticles/files/6966285/screencast-translate.google.com-2021.08.11-09_17_18.zip)

As you can see if I resize the window new particles appear from the right border. I want new particles to appear in the center of canvas. Help me once again.

Thanks.

**Reply by matteobruni** [reply]
You can disable the `number.density` options, it adjusts the particles number for the window size, but if you are using an emitter for adding particles is not necessary.

The `shape` section must be inside the `particles` options and not outside of it, if using `circle` is not necessary to add it since it's the default value.

Anyway, you can checkout also this sample: https://codepen.io/matteobruni/pen/GRpYdGv

It uses a javascript function for adding particles, each particle can be added with different options, if you want different move speed it can be achieved with this.

The function for adding particles is useful for particular configurations that are not supported yet. I was thinking about creating an out mode that make particles spawn from a specified area but it's just an idea for now

**Comment by returntowholeness** [comment]
@matteobruni I tried to add an image to the background class property of tsParticles but it doesn't show.  It appears for a half a second and the canvas. Is it possible to add a background image?

Here's the property.
https://particles.js.org/docs/classes/Options_Classes_Background_Background.Background.html#image

**Reply by matteobruni** [reply]
Please open your own discussion, yes it works except if you are using move trails since the fill color overrides the image.
